content/geodesic/kops/upgrade-cluster.md

```diff
@@ -13,23 +13,25 @@ RUN make -C /packages/install kops KOPS_VERSION=1.9.0
 
 Find the latest release of kops [here](https://github.com/kubernetes/kops/releases).
 
-Then follow the instructions for upgrading kubernetes.
+Then follow the instructions for upgrading kubernetes below.
 
 {{% dialog type="info" icon="fa fa-book" title="Read More" %}}
 <https://github.com/kubernetes/kops/blob/master/docs/releases.md>
 {{% /dialog %}}
 
 # Upgrading Kubernetes Release
 
-Upgrading `kops` cluster to a new release of kubernetes requires that we upgrade the `kubectl` client as well as select a kubernetes release that is compatible with kops.
+Upgrading `kops` cluster to a new release of kubernetes requires that we upgrade the `kubectl` client in `geodesic` as well as select a kubernetes release that is compatible with kops. Note, `kops` usually lags behind in support for the latest release of Kubernetes. If the latest release of `kops` is `v1.9`, then that means it supports up to kubernetes `v1.9` and will not be compatible with `v1.10` or newer.
 
 To explictly upgrade `kubectl`, add this to the `Dockerfile`
+
 ```
 RUN make -C /packages/install kubectl KUBECTL_VERSION=1.10.0
 ```
- The `kubectl` release version corresponds to the kubernetes release. Find the latest release of kubernetes [here](https://github.com/kubernetes/kubernetes/releases).
 
- Then follow the [official instructions](https://github.com/kubernetes/kops/blob/master/docs/upgrade.md) for upgrading kops.
+The `kubectl` release version corresponds to the kubernetes release. Find the latest release of kubernetes [here](https://github.com/kubernetes/kubernetes/releases).
```

Comment:
let's capitalize Kubernetes and use it consistently

Reply:
https://github.com/cloudposse/docs/issues/179